### DISCUSSION TITLE
Concat fields or add rows

### DISCUSSION THREAD

**Discussion by Tam3000** [abordage/nova-table-card#5] [Ideas]
Great job that greatly simplifies the use of cards in Nova 4.

Isn't it possible to concatenate two fields, for example in the title? I would like to be able to show first name + last name in the title. Or add a new row in card?

**Comment by abordage** [comment]
Hi, @Tam3000!
You can pass any string to the title, like so:

```php
$users = \App\Models\User::limit(5)->get();
foreach ($users as $user) {
    $rows[] = [
        'title' => $user->first_name . ' ' . $user->last_name,
        'viewUrl' => $this->getResourceUrl($user),
    ];
}
```

**Comment by Tam3000** [comment]
Great! Thanks
